services/checkin/service/checkin_service.go

```diff
@@ -69,8 +70,14 @@ func UpdateUserCheckin(id string, user_checkin models.UserCheckin) error {
 	return err
 }
 
+const (
+	SponsorRole = "Sponsor"
```

Comment:
Copy over this file https://github.com/HackIllinois/api/blob/master/services/auth/models/roles.go into the models folder for this service and use that.